WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -624,23 +624,20 @@ public void AttackSpeed(AttackSkill skill, AttributeSet attrs)
                     if (incAS != 0)
                         APS = IncreaseValueByPercentage(APS, incAS);
 
-                    float moreAS = 0;
+                    float moreAS = 1;
                     if (attrs.ContainsKey("#% more Attack Speed"))
-                        moreAS += attrs["#% more Attack Speed"][0];
+                        moreAS *= attrs["#% more Attack Speed"][0] / 100 + 1;
                     if (attrs.ContainsKey("#% less Attack Speed"))
-                        moreAS -= attrs["#% less Attack Speed"][0];
+                        moreAS /= attrs["#% less Attack Speed"][0] / 100 + 1;
```

Comment:
I'm pretty sure this should be `moreAS *= 1 - attrs[...]/100;`.
100% less attack speed means you never attack, not that you attack half as often.